GitCommands/Settings/AppSettings.cs

```diff
@@ -1003,6 +1004,8 @@ public static bool DontConfirmCommitIfNoBranch
             set => SetBool("DontConfirmCommitIfNoBranch", value);
         }
 
+        public static readonly ISetting<bool> DontConfirmBranchCheckout = Setting.Create(ConfirmationsSettingsPath, nameof(DontConfirmBranchCheckout), true);
```

Comment:
> This will make this setting separate from similar settings though

Yes, almost all settings in AppSettings.cs are named as negative, but ok, I’ll make the changes as you suggested.

Reply:
Yes, the old settings names have been kept for downward compatibility with existing setting files of the users.